python/kvikio/kvikio/_lib/CMakeLists.txt

```diff
@@ -17,10 +17,20 @@ set(cython_modules arr.pyx buffer.pyx defaults.pyx driver_properties.pyx file_ha
                    libnvcomp.pyx libnvcomp_ll.pyx
 )
 
+if(TARGET CURL::libcurl)
+  message(STATUS "Building remote_handle.pyx (libcurl found)")
+  list(APPEND cython_modules remote_handle.pyx)
+else()
+  message(
+    WARNING
+      "Skipping remote_handle.pyx (please set KvikIO_REMOTE_SUPPORT=ON for remote file support)"
+  )
```

Comment:
As with the previous discussions/issues around noisiness, should we actually warn here? Or just emit a `STATUS` message?

Reply:
Good point, done